openid-4-verifiable-presentations-1_0.md

```diff
@@ -261,6 +268,12 @@ This specification defines the following new parameters:
 
 A public key to be used by the Wallet as an input to the key agreement to encrypt Authorization Response (see (#jarm)). It MAY be passed by the Verifier using the `jwks` or the `jwks_uri` claim within the `client_metadata` or `client_metadata_uri` request parameter.
 
+`request_uri_method`: 
+: OPTIONAL. A string determining the HTTP method to be used with the `request_uri` included in the same request. Two case-insensitive values are defined for `request_uri_method` in this specification: `get` and `post`. If `request_uri_method`  is `get`, the Wallet MUST send the request to retrieve the Request Object using the HTTP GET method, i.e., as defined in [@RFC9101]. If `request_uri_method`  is `post`, a supporting Wallet MUST send the same request using the HTTP POST method as detailed in (#request_uri_method_post). 
```

Comment:
It's kind of implied but maybe worth explicitly saying that `get` is the default value assumed when/if `request_uri_method` isn't present?

Reply:
I think that's actually said (albeit a little indirectly but describing how the absence of `request_uri_method` is handled) in the next sentence.

Reply:
Yeah, that's what I meant when saying it's kind of implied. I suppose that's good enough. I'll "resolve conversation"